### PR TITLE
Movie detailss

### PR DESCRIPTION
**Pull request by Marianna-Weychan**
Okay team!
I created a MovieDetails page, so far without logic, I will add it when we have the rest of the necessary components.

### CODE REVIEW

**Comment by ghost** [comment]
I see that there is still a conflict in the App. I will probably be ok when Adrian approves the master :)

**Comment by KatarzynaZaloba** [comment]
Nice!
image

File: src/common/GlobalStyles.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ html {
 body {
     font-family: 'Poppins', sans-serif;
     font-weight: 400;
+    padding: 16px;
```
Comment (KatarzynaZaloba) [inline src/common/GlobalStyles.js:17]: When you delete the padding then you delete this little space around the image with the title:
image

Comment (KatarzynaZaloba) [reply src/common/GlobalStyles.js:17]: See?
image

**Review by KatarzynaZaloba — APPROVED** [review]

**Review by AdrianJar — APPROVED** [review]
This is one hell of a good job!

**Comment by ghost** [comment]
Kasia do you think that's enough?
So the problem is with the padding?
I've already tried with padding and margin but there's nothing stopping me from trying :D
Thanks for the advice :*

**Comment by ghost** [comment]
aaa ok now i see your screen :)
Is this a low resolution view?

**Pull request merged**